staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -577,7 +577,13 @@ func (s *SecureServingInfo) HostPort() (string, int, error) {
 // AuthorizeClientBearerToken wraps the authenticator and authorizer in loopback authentication logic
 // if the loopback client config is specified AND it has a bearer token.
 func AuthorizeClientBearerToken(loopback *restclient.Config, authn *AuthenticationInfo, authz *AuthorizationInfo) {
-	if loopback == nil || authn == nil || authz == nil || authn.Authenticator == nil && authz.Authorizer == nil || len(loopback.BearerToken) == 0 {
+	if loopback == nil || len(loopback.BearerToken) == 0 {
+		// do noting if either the loopback client config is nil or it doesn't have a bearer token.
```

Comment:
nothing

Reply:
the comment does not add anything to the if clause, does it?

Reply:
mmm. yeah. removed the comment